apps/user/endpoints/v1/user.py

```diff
@@ -0,0 +1,67 @@
+'''
+Path functions for /apps/user/v1/user
+'''
+
+from fastapi import status, APIRouter, Depends
+from fastapi.responses import JSONResponse
+
+from services.mongo_service import get_mongo
+from ...services.auth_service import auth_user
+from ...models import auth
+from ...models.message import Message
+
+router = APIRouter()
+
+
+@router.get(
+    '/',
+    dependencies=[Depends(auth_user)],
+    response_model=Message,
+    responses={
+        status.HTTP_401_UNAUTHORIZED: {'model': Message},
+        status.HTTP_404_NOT_FOUND: {'model': Message},
+    },
+)
+def get_user(
+    mail: str | None = None,
+    mongo = Depends(get_mongo),  # MongoDB database
+):
+    ''' Check if there's already the user in the DB. '''
+
+    # Check if there's already the same user.
+    if mongo.user.find_one({'mail': mail}):
```

Comment:
How about not using mongo directly but implementing these features in mongo_service.py? Changing mongo variable name to database can be great when we may need to change the db type in the future to sql or so on.
Such as `db = Depends(get_mongo)` and `if db.find_one(user, {'main': mail})` and implement the find_one function in mongo_service.py to do mongo.user.find_one({'mail': mail})

Reply:
I'll change it as you suggested.